backend/vkt/src/main/java/fi/oph/vkt/api/dto/clerk/ClerkEnrollmentMoveDTO.java

```diff
@@ -0,0 +1,12 @@
+package fi.oph.vkt.api.dto.clerk;
+
+import javax.validation.constraints.NotNull;
+import lombok.Builder;
+import lombok.NonNull;
+
+@Builder
+public record ClerkEnrollmentMoveDTO(
+  @NonNull @NotNull Long id,
+  @NonNull @NotNull Integer version,
+  @NonNull @NotNull Long toExamSessionId
```

Comment:
Olisko parempi nimi `newExamEventId` ?

Reply:
Sen taisin ensimmäiseksi itsekin kirjoittaa, mutta eihän tässä ole uudesta kysymys vaan mihin. Ihan sama, pidän näin, vaihdan jos nimen omaan haluat.

Reply:
Voi jättää näinkin tai sitten `toExamEventId`. Exam session tullut yki:ssä käytettyä muttei ainakaan toistaiseksi täällä?